mathics/builtin/inout.py

```diff
@@ -1820,6 +1820,7 @@ def apply_tex(self, expr, evaluation):
             # Replace multiple newlines by a single one e.g. between asy-blocks
             tex = MULTI_NEWLINE_RE.sub('\n', tex)
 
+
```

Comment:
I notice that, but I do not know how to ask git to revert just a file, or ignore this kind of changes.